### ISSUE TITLE
Incompatible with MacOS Big Sur

### ISSUE THREAD

**Issue by jclin22009** [fikovnik/ShiftIt#314]

**Comment by Ruko2010** [comment]
Same here. To describe the Problem a bit more. After Starting Shiftit the Window Appears that reminds me, that Shiftit needs "Accessibility" Access to work. However, this Access is already givven.
![image](https://user-images.githubusercontent.com/2239440/99083588-086c4e00-25c6-11eb-800b-cf68dc61452d.png)
![image](https://user-images.githubusercontent.com/2239440/99083679-28037680-25c6-11eb-8ba7-038427324b02.png)

**Comment by pallas** [comment]
There is another dialog from the system that pops up (I wasn't able to screen capture it) asking if ShiftIt should be allowed to control System Preferences. It's also not possible to uncheck the box in Accessibility on my system.

**Comment by ferranabello** [comment]
Hi guys, there is an easy fix to this. Just remove Shiftit using the "-" button and then add it again. After that, Shiftit will just work!

**Comment by jclin22009** [comment]
Yep, ferranabello's solution worked for me. Thanks!

**Comment by pallas** [comment]
Wish I would have seen this earlier, I just ended up installing Rectangle instead.

**Comment by niralis13** [comment]
Yes ferranabello it worked for me!! Thank you.
